### PR TITLE
Enable rspec multiple describe linter

### PR DESCRIPTION
**Pull request by kessio**
NOTE: Please review the pull request process before opening your first PR: https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/CONTRIBUTING.md#pull-request-process

## What this PR does
< describe the purpose of this pull request and note the issue it addresses >

**Enable Rspec/MultipleDescribe linter rule**
Fixed  Rspec/MultipleDescribe linter error by changing the feature specs for invalid and expired OAuth credentials to use a single describe block. The code has been organized into separate describe blocks, each containing one or more it blocks with a specific test case. This improves the readability and maintainability of the code. No functionality has been changed, only the structure of the code.

## Screenshots
Before:
< add a screenshot of the UI before your change >

After:
< add a screenshot of the UI after your change >
## Open questions and concerns
< anything you learned that you want to share, or questions you're wondering about related to this PR >

### CODE REVIEW

File: app/services/update_course_stats.rb [hunk 1]
```diff
@@ -6,9 +6,9 @@
 require_dependency "#{Rails.root}/lib/data_cycle/update_logger"
 require_dependency "#{Rails.root}/lib/analytics/histogram_plotter"
 require_dependency "#{Rails.root}/lib/importers/revision_score_importer"
-require_dependency "#{Rails.root}/lib/importers/average_views_importer"
-require_dependency "#{Rails.root}/lib/errors/update_service_error_helper"
-require_dependency "#{Rails.root}/lib/data_cycle/course_queue_sorting"
+require_dependency Rails.root.join('lib', 'importers', 'average_views_importer')
```
Comment (ragesoss) [inline app/services/update_course_stats.rb:9]: This seems unrelated.

Comment (kessio) [reply app/services/update_course_stats.rb:9]: Let me fix it, thanks.

Comment (kessio) [reply app/services/update_course_stats.rb]: @ragesoss I have fixed it, and now only 2 files have the necessary changes.
